embassy-stm32/src/usart/buffered.rs

```diff
@@ -254,7 +254,17 @@ impl<'d, T: BasicInstance> BufferedUart<'d, T> {
     }
 
     pub fn set_config(&mut self, config: &Config) -> Result<(), ConfigError> {
-        reconfigure::<T>(config)
+        reconfigure::<T>(config)?;
+
+        T::regs().cr1().modify(|w| {
+            #[cfg(lpuart_v2)]
```

Comment:
Is it `#[cfg(any(lpuart_v2, usart_v4))] `?

Reply:
lpuart doesn't exist as a peripheral anymore, it's usart_v4.